mlflow/rfunc/backend.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import os
+import re
+import subprocess
+
+from mlflow.models import FlavorBackend
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri
+from mlflow.utils.file_utils import TempDir
+
+_logger = logging.getLogger(__name__)
+
+
+class RFuncBackend(FlavorBackend):
+    """
+    Flavor backend implementation for the generic R models.
+    Predict and serve locally models with 'crate' flavor.
+    """
+    version_pattern = re.compile("version ([0-9]+[.][0-9]+[.][0-9]+)")
```

Comment:
What does this aim to do?

Reply:
parse what version of Rscript is installed. It should be >= mlflow/R requirements.